### DISCUSSION TITLE
National Stock Assessment Author Archive

### DISCUSSION THREAD

**Discussion by Schiano-NOAA** [nmfs-ost/asar#227] [General]
Please use this issue page to request adding your name and affiliation to the archive of stock assessment authors across the U.S. This list is intended to be exhaustive, so we would like to add everyone involved across the country. There is no need to add any information other than your preferred first and last names as well as middle initial (if desired) along with you affiliation. For NOAA employees, please include the science center you are affiliated with along with the location of your office.  For non-NOAA employees, please list the name of your organization with its address, city, state, and postal code.

NOAA Example:
SEFSC-Beaufort

Non-NOAA Example:
ECS Federal
2750 Prosperity Ave. Fairfax, VA 22301

We appreciate your contributions to U.S. fisheries management and your active use of this tool!

**Comment by kellijohnson-NOAA** [comment]
Attached is a csv with authors, ignore next column, and address for the NWFSC, SWFSC, and collaborators. Let me know if I can provide anything else or this in a better format.
[authors_NWFSC.csv](https://github.com/user-attachments/files/16458909/authors_NWFSC.csv)

The information was generated using {sa4ss} and the following code
```
library(sa4ss)
data(authors, package = "sa4ss")
everything_nwfsc <- authors |>
  dplyr::mutate(
    address = purrr::map_chr(author, .f = sa4ss::get_affiliation)
  )
write.csv(x = everything_nwfsc, file = "authors_NWFSC.csv")
```

**Comment by kellijohnson-NOAA** [comment]
@Schiano-NOAA one thing to think about wrt an author's address is country. I know right now we are focused on domestic assessments but international collaborators with highly migratory species is pretty standard. And, in the future the assessment of sablefish may become cross-boundary. Feel free to make this a different issue.

**Comment by okenk** [comment]
I noticed that non-NOAA affiliated authors still have a "NOAA" in their affiliation. For example, this gets generated in the skeleton file:
```
  - name: 'Alison D. Whitman'
    affiliations:
      - name: 'NOAA Fisheries Oregon Department of Fish and Wildlife'
        address: '2040 Southeast Marine Science Drive'
        city: 'Newport'
        state: 'OR'
        postal-code: '97365'
```

"NOAA Fisheries" should not be appended. I don't see it in any of the affiliation csv files, so I am guessing the code must be appending "NOAA Fisheries" somewhere. We have many assessment coauthors unaffiliated with NOAA.

**Comment by Schiano-NOAA** [comment]
@okenk Thanks for bringing this to my attention! This is now fixed in the package.

**Comment by melissamonk-NOAA** [comment]
Requested changes:
Melissa Monk; middle initial H

Julia Coates: middle initial H
Julia's correct address (could be labeled as CDFW-SB)
California Department of Fish and Wildlife
Marine Region
1933 Cliff Drive, Suite 9
Santa Barbara, CA 93109

**Comment by Schiano-NOAA** [comment]
@melissamonk-NOAA The changes you mentioned are now live in `asar`. If you encounter any issues, feel free to leave a new issue or email me!
